pkg/bgpv1/agent/controller.go

```diff
@@ -27,8 +27,15 @@ import (
 	"github.com/cilium/cilium/pkg/option"
 )
 
-// minHoldTime represents the minimal BGP hold time duration
-const minHoldTime = 3 * time.Second
+const (
+	// minHoldTime represents the minimal BGP hold time duration
+	minHoldTime = 3 * time.Second
+	// DefaultPeerPort is the TCP port number of a CiliumBGPNeighbor when
+	// PeerPort is unspecified.
+	DefaultPeerPort uint16 = 179
```

Comment:
nit: this should go in bgpv1/types/bgp.go

Reply:
@harsimran-pabla commit a138728 moves this const to the types pkg. Thanks for the guidance.